spread.yaml

```diff
@@ -962,6 +969,9 @@ suites:
             distro_install_package snapd qemu qemu-utils genisoimage sshpass qemu-kvm cloud-image-utils ovmf kpartx xz-utils
             snap install ubuntu-image --classic
 
+            # Install the snapd built
+            dpkg -i "$SPREAD_PATH"/../snapd_*.deb
```

Comment:
Why do we need that on the host when testing core in nested?

Reply:
Because for example we use the snapd on the host when we create the image, when we inject snap-bootstrap into the rebuilt kernel snap initramfs for uc20 and for any fakestore operations that I will be proposing, so it's more convenient and expected IMHO to get snapd from the spread run on the host when preparing the image etc.

Reply:
Ah, I see, thanks for explanation!